packages/esm-service-queues-app/src/patient-search/visit-form/base-visit-type.component.tsx

```diff
@@ -56,19 +57,21 @@ const BaseVisitType: React.FC<BaseVisitTypeProps> = ({ onChange, visitTypes }) =
               labelText=""
             />
           )}
-
-          <RadioButtonGroup
-            className={styles.radioButtonGroup}
-            defaultSelected={defaultVisitType}
-            orientation="vertical"
-            onChange={onChange}
-            name="radio-button-group"
-            valueSelected={results?.length >= 1 && results[0].uuid}>
-            {results.map(({ uuid, display, name }) => (
-              <RadioButton key={uuid} className={styles.radioButton} id={name} labelText={display} value={uuid} />
-            ))}
-          </RadioButtonGroup>
         </>
+      )}
+
+      {results.length ? (
```

Comment:
This looks right - moving the search conditional around the results table, but _not_ around the actual search box itself.